spec/rails_helper.rb

```diff
@@ -46,6 +47,13 @@
 RSpec.configure do |config|
   # add `FactoryBot` methods
   config.include FactoryBot::Syntax::Methods
+  config.include RequestSpecHelper, type: :request
+  config.include SerializerSpecHelper, type: :serializer
+  config.include RSpec::Rails::ModelExampleGroup, type: :validator
+  config.include RSpec::Rails::RequestExampleGroup, type: :validator
+  config.include Shoulda::Matchers::ActiveModel, type: :validator
+  config.include Shoulda::Matchers::ActiveRecord, type: :validator
+  config.include Shoulda::Matchers::ActionController, type: :validator
```

Comment:
These were manually included in our validator tests since our validator contains validation code related to both models and controllers.